views/artistResults.hbs

```diff
@@ -0,0 +1,16 @@
+<h1>Spotify</h1>
+<div>
+    {{#each searchedArtist}}
+
+    <h2>
+        {{name}}
+    </h2>
+
+    <div>
+        <img src="{{images.[0].url}}">
+    </div>
+
```

Comment:
The amount of empty lines can be a bit less.
Empty lines can be used to separate "chunks" of HTML, that are unrelated. For example, a section of a page.
To me the entirety of HTML in this view is interrelated.